scripts/create-package/plopfile.ts

```diff
@@ -16,7 +16,7 @@ const v8ReferencePackages = {
   node: ['codemods'],
 };
 const convergedReferencePackages = {
-  react: ['react-menu'],
+  react: ['react-provider'],
```

Comment:
Currently only `@fluentui/react-provider` is using Griffel packages (#21360). As this script pickups versions from a `package.json` I switched it to use different package as a source.